apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/VedtaksvurderingRoute.kt

```diff
@@ -308,6 +310,16 @@ fun Route.samordningSystembrukerVedtakRoute(vedtakSamordningService: VedtakSamor
     }
 }
 
+fun Route.vedtakForEksternt(vedtakForEksterntService: VedtakForEksterntService) {
```

Comment:
Her er naming vanskelig for mange tolker eksternt som ting utenfor NAV. Kanskje du har noen innspill her @oyvindsh ?

Reply:
Sikkert ok for now